app/services/waste_carriers_engine/worldpay_xml_service.rb

```diff
@@ -79,7 +79,7 @@ def build_address(xml)
 
       address1 = [address.house_number, address.address_line_1].join(" ")
       address2 = address.address_line_2
-      postcode = address.postcode
+      postcode = address.postcode.presence || "00000"
```

Comment:
I believe we said in the story we would set this to `"UNKNOWN"` if the postcode was missing.

Has this just been missed, or is there a reason I'm not aware of to set it to `"00000"`?

Reply:
Oh sorry! My mistake. 
```suggestion
      postcode = address.postcode.presence || "UNKNOWN"
```